### ISSUE TITLE
On iPhone 12, Header works incorrectly.

### ISSUE THREAD

**Issue by cnhsttyle** [react-native-elements/react-native-elements#2657]
![Simulator Screen Shot - iPhone 12 - 2020-10-30 at 17 59 08](https://user-images.githubusercontent.com/15974698/97680321-db09a580-1ad9-11eb-839b-d8bd158d7ea3.png)

Up to iPhone 11, it responds well to safearea and works properly.
But My iPhone 12 Simulator, Header works incorrectly.

my package.json... "react-native-elements": "^2.3.2".
please help me

**Comment by JB1905** [comment]
This problem is connected with https://github.com/ovr/react-native-status-bar-height/issues/17

**Comment by pranshuchittora** [comment]
Thanks @JB1905

**Comment by Simon-TechForm** [comment]
https://github.com/ovr/react-native-status-bar-height/issues/17 has been fixed in v2.6.0 even though the issue isn't closed. 

Updating to this version should fix the header issue. We have been using the fix (with patch-package) for about 10 days without any problems.

**Comment by kashmiry** [comment]
Like _**Simon-TechForm**_ have mentioned, it has been fixed in v2.6.0.
I use Yarn, so I added a resolution to fix it temporary.
Add to package.json a resolution block if it does not exists.
````
"resolutions": {
    "react-native-status-bar-height": "2.6.0"
}
````
Then `yarn install`

**Comment by McFly78** [comment]
I just solved the issue reinstalling react-native-elements 2.3.2 to update react-native-status-bar-height to 2.6.0 version.

In package-lock-json you can see that for new react-native-elements installation the react-native-status-bar-height version that will be installed is the latest until 2.X.X

"react-native-elements": {
        ...
        "react-native-status-bar-height": "^2.5.0"
